scaffold/scripts/setup.sh

```diff
@@ -70,9 +70,20 @@ while true; do
     esac
 done
 
+### Select webserver
+while true; do
+    printf "Choose kind of project ([monolithic] or decoupled): "
+    read -r REPLY;
+    KIND_PROJECT=${REPLY:-monolithic}
```

Comment:
Naming is important, this should be `PROJECT_TYPE` and the values should be `regular` and `decoupled`, I personally don't like the term monolith since is ambiguous